examples_ch5xx/blink/blink.c

```diff
@@ -0,0 +1,35 @@
+#include "ch32fun.h"
+#include <stdio.h>
+
+// use defines to make more meaningful names for our GPIO pins
+#define PIN_1 PA4
+#define PIN_K PA5
+#define PIN_BOB PA8
+#define PIN_KEVIN PA9
+
+int main()
+{
+	SystemInit();
+
+	funGpioInitAll(); // Enable GPIOs
```

Comment:
Only thing I would add here is that depending on the chip, this may be a no-op.

Reply:
all of ch5xx is no-op, I'll add that

Reply:
Thanks!